test/rendering/cases/webgl-fill-pattern/main.js

```diff
@@ -104,6 +104,9 @@ new Map({
   }),
 });
 
-render({
-  message: 'renders four polygons with various pattern fills',
+requestAnimationFrame(() => {
+  map.renderSync();
+  render({
+    message: 'renders four polygons with various pattern fills',
+  });
```

Comment:
I'm wondering if it wouldn't be better to do something like this instead:
```js
map.once('rendercomplete', () => {
  render({
    message: 'renders four polygons with various pattern fills',
  });
});
```

Reply:
Amended the first commit with this change